app/authentication/login/login.component.e2e-spec.ts

```diff
@@ -0,0 +1,40 @@
+import { browser, by, element } from 'protractor'
+
+describe('Login', () => {
+
+  beforeEach(() => {
+    browser.get('/login');
+  });
+
+  it('should have an input and submit button', () => {
+
+    let allInputElements = element.all(by.css('md-card md-card-content form input'));
+    let inputElement = element(by.css('md-card md-card-content form input'));
+    let submitButton = element(by.css('md-card md-card-content form button'));
+    let loginCardTitle = element(by.css('md-card md-card-title h3'));
+
+    browser.wait(function () {
+      return inputElement.isPresent();
+    }, 60000);
+
+    expect(inputElement.isPresent()).toEqual(true);
+    expect(submitButton.isPresent()).toEqual(true);
+    expect(element(by.css('md-card md-card-title h3')).getText()).toEqual('Geben Sie Ihre Daten ein, um sich einzuloggen');
```

Comment:
the `h3` is in the current code a `p`. But it's enough to select the element by `md-card-title`